src/opengl_spectrum.cpp

```diff
@@ -357,6 +357,7 @@ bool CVisualizationSpectrum::Start(int iChannels, int iSamplesPerSec, int iBitsP
     for(y = 0; y < 16; y++)
     {
       m_cHeights[y][x] = 0.0;
+      m_heights[y][x] = 0.0;
```

Comment:
Why you used the master branch? Them is no more used, only the Kodi related "Leia" and "Matrix" are used. Thats also why "Matrix" as master is set.

This line change makes there sense on both versions, only the "src/shaders/Shader.cpp" is no more used.

Reply:
I was experimenting mainly with Kodi Leia; I've not checked yet with Matrix; I will check.